src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -348,7 +346,14 @@ private void readResources(Document document) {
               Name.anyCamel(request.getName(), "resource").toLowerUnderscore();
           input.getFields().add(new Field(requestFieldName, request, false, null, null));
           methodHttpOption.getProperties().put("body", requestFieldName);
+          methodSignatureParamNames.put("", requestFieldName);
```

Comment:
What does this mean?
(a) Are you putting non-required fields in the method signature
(b) What is the significance of the empty key `""`?

(Consider a one-sentence comment above this code stanza, ie if-statement)

Reply:
I explained it in the comment above `methodSignatureParamNames` variable initialization.